Makefile.PL

```diff
@@ -0,0 +1,11 @@
+use 5.024003;
+use ExtUtils::MakeMaker;
+
+WriteMakefile(
+    NAME          => 'DBIx::Result::Convert::JSONSchema',
+    VERSION_FROM  => 'lib/DBIx/Result/Convert/JSONSchema.pm',
+    PREREQ_PM     => {},
```

Comment:
Need to list non-core pre-reqs and versions here. Probably makes sense to list all modules required, core or non-core, which is what I usually do.